src/data/abilities/ab-attrs/ab-attr.ts

```diff
@@ -39,7 +39,14 @@ export abstract class AbAttr {
     return false;
   }
 
-  getTriggerMessage(_pokemon: Pokemon, _abilityName: string, ..._args: any[]): string | null {
+  /**
+   * @param pokemon - The {@linkcode Pokemon} with this ability
+   * @param abilityName - The name of the {@linkcode Ability} with this attribute
```

Comment:
```suggestion
   * @param abilityName - The localized name of the {@linkcode Ability} with this attribute
```
(just so it's clear that there's no need to worry about localization at this point)